packages/aws_mq/manifest.yml

```
@@ -1,7 +1,7 @@
 format_version: 3.3.1
 name: aws_mq
 title: "Amazon MQ"
-version: 0.5.0
+version: 0.6.0
 description: "Collect Amazon MQ metrics with Elastic Agent"
```

Should this be changed?

```suggestion
description: "Collect Amazon MQ metrics and logs with Elastic Agent"
```